views/theme/package.json

```diff
@@ -2,56 +2,57 @@
   "scripts": {
     "start": "webpack"
   },
-  "name": "geotix",
-  "version": "1.0.0",
-  "description": "A Geotix theme",
-  "author": "GeoTix, Inc.",
+  "name": "evvnt",
+  "version": "2.0.0",
+  "description": "Dynamic Theme for the Evvnt One Platform",
+  "author": "Evvnt",
+  "type": "commonjs",
   "license": "ISC",
   "dependencies": {
-    "@material/animation": "^0.41.0",
-    "@material/base": "^0.41.0",
-    "@material/card": "^0.43.0",
-    "@material/chips": "^0.43.0",
-    "@material/dialog": "^0.43.0",
-    "@material/drawer": "^0.44.1",
-    "@material/elevation": "^0.43.0",
-    "@material/fab": "^0.43.0",
-    "@material/floating-label": "^0.43.0",
-    "@material/form-field": "^0.43.0",
-    "@material/icon-toggle": "^0.43.0",
-    "@material/layout-grid": "^0.41.0",
-    "@material/line-ripple": "^0.43.0",
-    "@material/linear-progress": "^1.0.0",
-    "@material/list": "^0.43.0",
-    "@material/menu": "^0.43.0",
-    "@material/menu-surface": "^0.43.0",
-    "@material/notched-outline": "^0.43.0",
-    "@material/radio": "^0.43.0",
-    "@material/ripple": "^0.43.0",
-    "@material/select": "^0.43.0",
-    "@material/slider": "^0.43.0",
-    "@material/snackbar": "^0.43.0",
-    "@material/switch": "^0.43.0",
-    "@material/tab-bar": "^0.43.0",
-    "@material/tab-scroller": "^0.43.0",
-    "@material/textfield": "^0.43.0",
-    "@material/theme": "^0.41.0",
-    "@material/toolbar": "^0.43.0",
-    "@material/top-app-bar": "^0.44.1",
-    "@material/typography": "^0.43.0",
+    "@material/animation": "^3.1.0",
+    "@material/base": "^3.1.0",
+    "@material/card": "^3.1.0",
+    "@material/chips": "^3.1.0",
+    "@material/dialog": "^3.1.0",
+    "@material/drawer": "^3.1.0",
+    "@material/elevation": "^3.1.0",
+    "@material/fab": "^3.1.0",
+    "@material/floating-label": "^3.1.0",
+    "@material/form-field": "^3.1.0",
+    "@material/icon-toggle": "^0.44.1",
+    "@material/layout-grid": "^3.1.0",
+    "@material/line-ripple": "^3.1.0",
+    "@material/linear-progress": "^3.1.0",
+    "@material/list": "^3.1.0",
+    "@material/menu": "^3.1.0",
+    "@material/menu-surface": "^3.1.0",
+    "@material/notched-outline": "^3.1.0",
+    "@material/radio": "^3.1.0",
+    "@material/ripple": "^3.1.0",
+    "@material/select": "^3.1.0",
+    "@material/slider": "^3.1.0",
+    "@material/snackbar": "^3.1.0",
+    "@material/switch": "^3.1.0",
+    "@material/tab-bar": "^3.1.0",
+    "@material/tab-scroller": "^3.1.0",
+    "@material/textfield": "^3.1.0",
+    "@material/theme": "^3.1.0",
+    "@material/toolbar": "^2.3.0",
+    "@material/top-app-bar": "^3.1.0",
+    "@material/typography": "^3.1.0",
     "mdl-stepper": "^1.1.6"
   },
   "devDependencies": {
-    "@material/button": "^0.43.0",
-    "css-loader": "^2.1.0",
+    "@material/button": "^3.1.0",
```

Comment:
this should be up in `dependencies`, right? seems weird that it was ever only in `devDependencies`.

Reply:
Yeah, that's odd. Will address